packages/content-management/favorites/favorites_public/src/components/favorites_empty_state.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { FormattedMessage } from '@kbn/i18n-react';
+import { i18n } from '@kbn/i18n';
+import { EuiEmptyPrompt, useEuiTheme, EuiImage } from '@elastic/eui';
+import { css } from '@emotion/react';
+import React from 'react';
+import emptyFavoritesDark from './empty_favorites_dark.svg';
+import emptyFavoritesLight from './empty_favorites_light.svg';
+
+export const FavoritesEmptyState = ({
+  emptyStateType = 'noItems',
+  entityNamePlural = i18n.translate('contentManagement.favorites.defaultEntityNamePlural', {
+    defaultMessage: 'items',
+  }),
+  entityName = i18n.translate('contentManagement.favorites.defaultEntityName', {
+    defaultMessage: 'item',
+  }),
+}: {
+  emptyStateType: 'noItems' | 'noMatchingItems';
+  entityNamePlural?: string;
+  entityName?: string;
+}) => {
+  const title =
+    emptyStateType === 'noItems' ? (
+      <FormattedMessage
+        id="contentManagement.favorites.noFavoritesMessageHeading"
+        defaultMessage="You haven’t starred any {entityNamePlural}"
+        values={{ entityNamePlural }}
+      />
+    ) : (
+      <FormattedMessage
+        id="contentManagement.favorites.noMatchingFavoritesMessageHeading"
+        defaultMessage="No starred {entityNamePlural} match your search"
```

Comment:
```suggestion
        defaultMessage="No starred {entityNamePlural} match your search."
```
Missing punctuation compared to the regular version of this empty state message

Reply:
I think we don't need `.` in the heading, or do we? 

The regular state is not visually heading, but I'd rather removed `.` there also

Reply:
Just to make sure, this is what I was referring to: 
<img width="369" alt="image" src="https://github.com/user-attachments/assets/07153969-7f17-4131-bb5f-baa89252fa9f">

If it looks like this, we need ending punctuation
If it's an alternate heading for the empty state, we don't need punctuation you are correct 👍